### PR TITLE
lib/os.h, test/util.c: define _USE_MATH_DEFINES for windows builds

### PR DESCRIPTION
**Pull request by sezero**
it is required to access the M_PI macro.

### CODE REVIEW

**Comment by sezero** [comment]
PING?

**Comment by uthidata** [comment]
I'm not a dev or maintainer so I'm just going to link the related issue here https://github.com/xiph/vorbis/issues/75

**Comment by sezero** [comment]
PING?

**Comment by sezero** [comment]
PING?

File: lib/os.h [hunk 1]
```diff
@@ -20,6 +20,9 @@
 #include "config.h"
 #endif
 
+#ifdef _WIN32
```
Comment (rillian) [inline lib/os.h:23]: Is `_WIN32` what we want here, or would _MSC_VER be better? mingw probably acts like gcc?

cc @evpobr

**Review by rillian — COMMENTED** [review]
Thanks for the fix.

**Comment by rillian** [comment]
I [submitted](https://gitlab.xiph.org/xiph/vorbis/-/merge_requests/19) a merge request with your patch rewritten to use the `_MSC_VER` conditional. It [seems to be green](https://ci.appveyor.com/project/rillian/vorbis/builds/39077466).

File: lib/os.h (hunk 1 above)
Comment (sezero) [reply lib/os.h:23]: For _MSC_VER: definitely needed. Harmless for others.

Comment (rillian) [reply lib/os.h:23]: Right, although the documented intention is slightly different.

**Comment by rillian** [comment]
Fixed instead by copying the fallback define from os.h in 894d1b1f22f2fb8ab6d3aeba863e143416c71ce2.
https://gitlab.xiph.org/xiph/vorbis/-/merge_requests/20

**Pull request closed without merging**